pkg/storage/s3/s3.go

```diff
@@ -446,6 +446,10 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 						Key:   aws.String("kubernetes.io/cluster/" + infra.Status.InfrastructureName),
 						Value: aws.String("owned"),
 					},
+					{
```

Comment:
We don't need this tag in s3 bucket for deletion.. also terraform is NOT uses for deleltion